packages/multichain/src/adapters/caip-permission-adapter-permittedChains.ts

```diff
@@ -45,7 +56,7 @@ export const getPermittedEthChainIds = (
 export const addPermittedEthChainId = (
   caip25CaveatValue: Caip25CaveatValue,
   chainId: Hex,
-) => {
+): Caip25CaveatValue => {
```

Comment:
Thanks for this! I keep meaning to add the lint rule requiring return types. It's a documented best practice already in our docs.